automatic/kitty/tools/chocolateyInstall.ps1

```diff
@@ -1,11 +1,32 @@
-﻿$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+. $toolsPath\helpers.ps1
+
+StopProcesses
+
+$packageArgs = @{
+    PackageName    = 'kitty'
+    FileFullPath   = Get-Item $toolsPath\kitty-bin-0.76.0.8.zip
```

Comment:
The call to `Get-Item` should not be here anymore. However, I'll take care of it when merging it in.